crypto/ake/enclave/src/lib.rs

```diff
@@ -166,9 +166,10 @@ impl<EI: EnclaveIdentity> AkeEnclaveState<EI> {
         let mut csprng = McRng::default();
         // TODO: Cache expected values rather than creating a new report each time.
         let report_body = Report::new(None, None)?.body();
+        let expected_measurements: [Measurement; 1] = [report_body.mr_enclave().into()];
         let initiator = Start::new(
             peer_id.to_string(),
-            report_body.mr_enclave(),
+            &expected_measurements.to_vec(),
```

Comment:
Ah, yes, *this* doesn't need to be a `to_vec()`, it should just be `&expected_measurements[..]`